discord-bot/deploy-commands.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { Routes } = require('discord.js');
+const { REST } = require('@discordjs/rest');
+const { clientId, guildId, token } = require('./config.json');
```

Comment:
@vkarpov15 The discord 'server' and 'app' details are expected in a `config.json` file as per this line. As per `.env.example` mentioned in the README, those should be available under `.env`. So, either this needs to be changed to `process.env.****` or the README needs to be updated.

Reply:
Good catch, thanks. `config.json` was what this sample used to use for config, I switched to `.env` because that is a more common pattern, particularly for Express applications.